### ISSUE TITLE
`covidcast::aggregate_signals` option to retain `issue` column?

### ISSUE THREAD

**Issue by capolitsch** [cmu-delphi/covidcast#269]
When using the `covidcast::aggregate_signals` function to convert a `covidcast_signal` data frame from long to wide format it would be great if the output retained (or had an option to retain) an `issue` column that specifies the latest issue date of the signal value. This would be convenient for, e.g., internal forecaster filtering w.r.t. a chosen backfill buffer.

**Comment by capnrefsmmat** [comment]
Yes, I'd like it to retain `issue` and all other columns from the original data, if possible, so one can switch between wide and long formats easily.

**Comment by chinandrew** [comment]
If you aggregate two signals and they have different issue dates, what should expected output be?

**Comment by capnrefsmmat** [comment]
In wide format, I think we'd need an issue column per signal, rather than one "global" issue column.
